scripts/registry.sh

```diff
@@ -91,6 +90,18 @@ publish() {
       | (popd > /dev/null && git hash-object -w --stdin))
 
     git commit -am "chore: update versions"
+
+    while ! yarn lerna publish from-package \
+      --dist-tag="$DISTTAG" --no-git-reset --no-verify-access --yes; do
+      echo 1>&2 "Retrying publish..."
+      sleep 5
+    done
+
```

Comment:
@rabi-siddique Have we tested it with multiple re-triggering of job? Did we see any improvement?
Can check logs if we actually see `Retrying publish...` in stderr.

Reply:
After implementing this change, I haven’t experienced any flakiness or failures in the `Start local NPM registry` step throughout all the testing I’ve done so far in the integration tests workflow.

I did encounter an instance of `Retrying publish...` during a workflow run, but the step did not fail, and `yarn lerna publish` was successfully executed again.

Reply:
Sounds good 👍